doc/workflows_and_tools/tools/index.rst

```diff
@@ -31,6 +31,303 @@ a glimpse in the connection to your user store, you could as well in a first
 step *mark* the orphaned tokens. A day later you could run the script again
 and delete those tokens, which are (still) *orphaned* and *marked*.
 
+With version 3.7 it can also filter for token attributes and attribute values.
+It is also possible to check just for the existence or not-existence of a
+certain tokeninfo-value.
+
+
+Find
+~~~~
+
+With the token-janitor you have the possibility to search for tokens in different ways.
+
+Description
+***********
+Searches through all tokens and returns the ones with the selected description.
+
+Example::
+
+    privacyidea-token-janitor find --description ^fo.*
```

Comment:
This does not work out!
Please note, that you need to enclose the asterix in the command line!

     privacyidea-token-janitor find --description '^fo.*'

otherwise the shell/bash will extend the "*".